pkg/tarmak/kubectl/kubectl.go

```diff
@@ -190,6 +190,14 @@ func (k *Kubectl) ensureWorkingKubeconfig() (interfaces.Tunnel, error) {
 
 	// check if certificates are set
 	if len(authInfo.ClientCertificateData) == 0 || len(authInfo.ClientKeyData) == 0 || len(cluster.CertificateAuthorityData) == 0 {
+
+		if err := k.tarmak.Terraform().Prepare(k.tarmak.Environment().Hub()); err != nil {
+			return nil, fmt.Errorf("failed to prepare terraform: %s", err)
+		}
+
+		// write ssh_config
+		k.tarmak.SSH().WriteConfig()
```

Comment:
Why are we doing that here again, it is already in https://github.com/jetstack/tarmak/blob/master/pkg/tarmak/tarmak.go#L310

If there is a good reason for it please add it to the comments